apps/foundation-demo/src/app/cart/components/cart/cart.component.scss

```diff
@@ -29,8 +20,13 @@
     font-size: $small-font-size;
 
     @include breakpoint(large) {
-      margin-left: 25px;
+      margin-left: 15px;
       float: none;
```

Comment:
Let's get rid of this float too. Should be able to with `grid` and `justify-self: right`

Reply:
fixed with `display: flex`